.github/workflows/release.yml

```diff
@@ -0,0 +1,85 @@
+name: Build Release
```

Comment:
hey man,
I've seen that you removed the former github action but I am not seeing the cargo publish step.